LiveSplit.AutoSplitters.xml

```diff
@@ -15566,32 +15566,34 @@
     </AutoSplitter>
     <AutoSplitter>
         <Games>
-            <Game>Red Ball</Game>
+            <Game>Red Ball 5</Game>
+            <Game>Red and Blue Balls</Game>
+            <Game>Red and Blue Balls 2</Game>
+            <Game>Red and Blue Balls 3</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rb1improved.asl</URL>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/RedBallGames.asl</URL>
         </URLs>
         <Type>Script</Type>
         <Description>Perfectly splits upon completing each level. Click "Website" to download the game. (By Maximum)</Description>
         <Website>https://drive.google.com/file/d/12X0J05rZ6gRnPgybDJuz56xGGnxjaOev/view?usp=sharing</Website>
     </AutoSplitter>
     <AutoSplitter>
         <Games>
+            <Game>Red Ball</Game>
             <Game>Red Ball 2</Game>
             <Game>Red Ball 3</Game>
             <Game>Red Ball 4 Vol.1</Game>
             <Game>Red Ball 4 Vol.2</Game>
             <Game>Red Ball 4 Vol.3</Game>
-            <Game>Red Ball 5</Game>
-            <Game>Red and Blue Balls</Game>
-            <Game>Red and Blue Balls 2</Game>
-            <Game>Red and Blue Balls 3</Game>
+            <Game>Multiple Red Ball Games</Game>
+            <Game>Red Ball Category Extensions</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/RedBallGames.asl</URL>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rbFullSeries.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Splits, starts, and resets. Click "Website" for the right game version and the README. (By Maximum, Ero)</Description>
```

Comment:
What's the reasoning for removing an author here?

Reply:
I do not care.